pyproject.toml

```diff
@@ -176,3 +179,6 @@ omit = ["**/venv/*", "**/env/*", "**/virtualenvs/*", "**/node_modules/*", "**/mi
 DJANGO_SETTINGS_MODULE = "brainwave.settings.local_base"
 pythonpath = [".", "backend"]
 python_files = ["test_*.py", "*_test.py", "tests/*.py"]
+
+[tool.pyright]
+exclude = ["**/tests","**/migrations", "**/models.py"]
```

Comment:
These are files that are really hard to type check, but really aren't worth the effort (at least for now)